server/views/serviceProviderReferrals/actionPlanConfirmation.njk

```diff
@@ -0,0 +1,30 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% extends "../partials/layout.njk" %}
+
+{% set pageTitle = "HMPPS Interventions" %}
+{% block pageTitle %}
+  {{ pageTitle }}
```

Comment:
I think @tomsmyers pointed this out a few weeks ago, but is there a reason we set the page title to a variable like this and then use it straight away, or is it just copied from other templates?

Reply:
Copied brainlessly 😄 - I've got rid of the dance.